libs/explorers/comparison-tools/src/lib/comparison-tool-table-link/comparison-tool-table-link.component.ts

```diff
@@ -12,7 +12,20 @@ export class ComparisonToolTableLinkComponent {
   linkText = input<string>('');
   linkUrl = input<string>('');
 
-  isExternalLink() {
-    return isExternalLink(this.linkUrl());
+  get internalLinkUrl(): string {
+    const url = this.linkUrl().trim();
+
+    // Return URL as-is if it already starts with '/'
+    if (url.startsWith('/')) {
+      return url;
+    }
+
+    // Prepend '/' to make it a proper internal route
+    return `/${url}`;
+  }
```

Comment:
Could consider using the `computed` signal here instead, since we're using `input` signals above, e.g. something like this: 

```suggestion
  internalLinkUrl = computed(() => {
    const url = this.linkUrl().trim();

    // Return URL as-is if it already starts with '/'
    if (url.startsWith('/')) {
      return url;
    }

    // Prepend '/' to make it a proper internal route
    return `/${url}`;
  });
```

Reply:
Good idea.  Fixed in next commit.